src/Components/SingleArticle.jsx

```diff
@@ -1,14 +1,23 @@
 import { useEffect, useState } from "react";
 import { getSingleArticleFromApi } from "./Utils/api";
 import { useParams } from "react-router-dom";
-import ArticleByTopic from "./ArticleByTopic";
-
+import { increaseVotes } from "./Utils/api";
 export const SingleArticle = () => {
+  const clickHandler = (e, increment) => {
```

Comment:
This should follow an optimistic rendering pattern ie once clicked:

- update UI
- make api call
- reverse UI change if an error is caught

We should not refetch data